main.go

```diff
@@ -501,12 +502,14 @@ func (a *app) walkType(source, sink, x string, m types.Type, w io.Writer, import
 
 }
 
+var cleanRE = regexp.MustCompile(`\W`)
```

Comment:
Wdyt about renaming this to something like `importSanitizerRE` to show where it is used?

Reply:
done